config/config.go

```diff
@@ -79,13 +79,14 @@ type files struct {
 
 //s3 is for configuring the s3 integration
 type s3 struct {
-	Enabled   bool   `yaml:"enabled"`
-	Endpoint  string `yaml:"endpoint"`
-	AccessKey string `yaml:"accessKey"`
-	Secret    string `yaml:"secret"`
-	Bucket    string `yaml:"bucket"`
-	Region    string `yaml:"region"`
-	ACL       string `yaml:"acl"`
+	Enabled         bool   `yaml:"enabled"`
+	Endpoint        string `yaml:"endpoint"`
+	ServingEndpoint string `yaml:"servingEndpoint"`
+	AccessKey       string `yaml:"accessKey"`
+	Secret          string `yaml:"secret"`
+	Bucket          string `yaml:"bucket"`
+	Region          string `yaml:"region"`
+  ACL             string `yaml:"acl"`
```

Comment:
Spacing issue 👀

Reply:
This is what I get for using the Web UI for merge 😶 I'll fix when I'm at my personal machine.